cpacker/MemGPT
# feat: Updated chat to display the currently selected agent

**Please describe the purpose of this pull request.**
The purpose of this pull request is to enhance the user interface of the chat component by displaying the name of the current agent. It adds a header (`<h1>`) element which shows the name of the active agent, or displays 'No Agent Selected' when no agent is currently chosen.

**How to test**
1. Navigate to the chat component in the UI.
2. Observe the header displaying the name of the current agent. If no agent is selected, it should display 'No Agent Selected'.
3. Test with different agents to ensure the name changes accordingly.
4. Verify the styling (centered text, font size, and weight) is consistent with the UI's design.

**Have you tested this PR?**
Yes, the latest commit on this PR has been tested. Upon rendering the chat component, the agent's name is correctly displayed at the top in a styled `<h1>` tag. When no agent is selected, it defaults to showing 'No Agent Selected'. The text is centered, and the font styling is in line with the existing UI design.

**Related issues or PRs**
N/A - This is a standalone update for improving the UI.

**Is your PR over 500 lines of code?**
No, this PR is a small update and only involves a single line of code addition to the chat component.

**Additional context**
N/A
